modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,108 @@
+
+---
+
+[NOTE]
+====
+This ability lets you keep detachments outside of the play area at the start of the game, similar to Reserves.
+You can then deploy them later in the game.
+When they arrive, you disembark them into the play area through a unit -- or objective -- that has this ability.
+====
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, as attributes in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit you can embark/disembark through that portal during the game.
+
+* `_(INF)_`: Infantry units.
+* `_(4+)_`: Vehicle units that have an Armour value of `4+` or lower.
+* `_({walker})_`: Vehicle units that have the {walker} ability, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that use it:
+
+* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.
+* `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.
+When a detachment arrives from Portal Reserves, you *must* disembark it from a {portal} unit/objective, if possible.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* When you wish to move a detachment into or out of a portal, you may choose only Advance or Assault orders.
+* When you wish to move a detachment into or out of a portal, use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the units.
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
+* You cannot use portals to transfer detachments *out* of the play area during the game.
+However, in the Movement phase only, you may move a detachment that is already in the play area into an available portal -- but then you *must* immediately disembark the detachment from a different available portal elsewhere in the play area.
```

Comment:
This ability is relatively complex so, inevitably, I'm finding little logical gaps and edge cases as I edit.

As it's for the Experimental section we can gloss some of these questions if we get to a point where we just want to get it out there -- but in the meantime I'll note them for consideration. In this case:

- What if some units have insufficient movement or are not permitted to pass through one of the two portals? Should we leave them behind, potentially out of command? Or should they be destroyed?

Reply:
We're following the standard embark/disembark rules, so they'll be left behind.